qiskit/providers/models/backendconfiguration.py

```diff
@@ -300,3 +340,115 @@ def __init__(self, backend_name, backend_version, n_qubits, basis_gates,
                          meas_lo_range=meas_lo_range, dt=dt, dtm=dtm,
                          rep_times=rep_times, meas_kernels=meas_kernels,
                          discriminators=discriminators, **kwargs)
+
+    @property
+    def dt(self) -> float:  # pylint: disable=invalid-name
+        """Time delta between samples on the signal channels in seconds."""
+        warnings.warn("The time delta is now returned in units of [s] rather than [ns].")
+        return self._dt * 1.e-9
+
+    @property
+    def dtm(self) -> float:
+        """Time delta between samples on the acquisition channels in seconds."""
+        warnings.warn("The time delta is now returned in units of [s] rather than [ns].")
+        return self._dtm * 1e-9
+
+    @property
+    def sample_rate(self) -> float:
+        """Sample rate of the signal channels in Hz (1/dt)."""
+        return 1.0 / self.dt
+
+    def hamiltonian(self) -> str:
```

Comment:
In the future this should return a Hamiltonian object. I think for now we should just return the Hamiltonian dictionary as this is the Hamiltonian "object". I believe @DanPuzzuoli will be working on doing a proper implementation moving forward.